libbrowser/src/libbrowser_uiwebview.mm

```diff
@@ -818,10 +818,19 @@ - (BOOL)webView:(UIWebView *)webView shouldStartLoadWithRequest:(NSURLRequest *)
 	
 	/* UNCHECKED */ MCCStringClone([request.URL.absoluteString cStringUsingEncoding: NSUTF8StringEncoding], t_url);
 	
-	if (!m_frame_request)
-		m_instance->OnNavigationBegin(false, t_url);
+	if ([NSURLConnection canHandleRequest: request])
+	{
+		if (!m_frame_request)
+			m_instance->OnNavigationBegin(false, t_url);
 
-	return YES;
+		return YES;
+	}
+	else
+	{
+		m_instance->OnNavigationRequestUnhandled(m_frame_request, t_url);
+		
+		return NO;
+	}
```

Comment:
I'm not sure your question makes sense in the context. This change is to the iOS UIWebViewDelegate - the delegate API differs from that of the OSX WebView delegates.